server/.env

```diff
@@ -1 +1,4 @@
-MONGO_URL = mongodb+srv://betty:ah1234@cluster0.nlcpvlz.mongodb.net/?retryWrites=true&w=majority
\ No newline at end of file
+MONGO_URL = mongodb+srv://betty:ah1234@cluster0.nlcpvlz.mongodb.net/?retryWrites=true&w=majority
```

Comment:
env. should not be pushed to git. 
Ask Sarah how to make env. file make git forget the env. file

Reply:
ok